system-test/bigtable.ts

```diff
@@ -1259,6 +1259,37 @@ describe('Bigtable', () => {
       assert.strictEqual(metadata.name, backupNameFromCluster);
       assert.deepStrictEqual(backup.expireDate, updateExpireTime);
     });
+
+    it.only('should get an Iam Policy for the backup', async () => {
+      const policyProperties = ['version', 'bindings', 'etag'];
+      const [policy] = await BACKUP.getIamPolicy();
+      console.log(`policy => \n${JSON.stringify(policy)}`);
+
+      policyProperties.forEach(property => {
+        assert.strictEqual(Object.keys(policy).includes(property), true);
```

Comment:
I think we'd want to test that `policyProperties.length` is always > 0, or confirm the inner asserts from some of these tests will run.

Reply:
How about 
```ts
      policyProperties.forEach(property => {
        assert(property in policy);
      });
```

Reply:
Sorry, I misread the code. I thought `policyProperties` was part of the response from the API, not a locally-defined array. Either way looks fine to me!